modules/assim.batch/R/pda.mcmc.bs.R

```diff
@@ -29,7 +29,7 @@ pda.mcmc.bs <- function(settings, params.id=NULL, param.names=NULL, prior.id=NUL
                   adj.min=adj.min, ar.target=ar.target, jvar=jvar, n.knot=n.knot, burnin=burnin)
  
    burnin <- ifelse(!is.null(settings$assim.batch$burnin), 
-                    settings$assim.batch$burnin, 
+                    as.numeric(settings$assim.batch$burnin), 
                     ceiling(min(2000,0.2*settings$assim.batch$iter)))
```

Comment:
the `burnin` was under postprocess function before, during the tests I moved it under separate functions and left them there but I think I can put it back now, so that it will be assigned once

`ceiling(min(2000,0.2*settings$assim.batch$iter))` was there before, if nothing is specified in the settings burnin is the 20% of the MCMC samples, @mdietze do you want this default to be changed?